rust/src/mqtt/mqtt.rs

```diff
@@ -497,12 +511,16 @@ impl MQTTState {
         return AppLayerResult::ok();
     }
 
-    fn parse_response(&mut self, input: &[u8]) -> AppLayerResult {
+    fn parse_response(&mut self, flow: *const Flow,stream_slice: StreamSlice) -> AppLayerResult {
+        let input = stream_slice.as_slice();
         let mut current = input;
+
         if input.is_empty() {
             return AppLayerResult::ok();
         }
 
+        let _pdu = Frame::new(flow, &stream_slice, input, input.len() as i64, MQTTFrameType::Pdu as u8);
```

Comment:
The suricata verify tests for PDU frame don't PASS as well.

Reply:
Can you share that test, too? It's ok to just push it to the draft PR you've submitted :)

Reply:
I have submitted the draft PR for suricata verify tests as well.

Reply:
I saw it, but you've mentioned tests for pdu frames here, while in the SV PR I could only find a test for the header, that's why I thought you may have other tests that had not been committed there...

Reply:
Oh Ok. I will push that too.

Reply:
done